sdk/storage/azblob/CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 * Added support for [Cold tier](https://learn.microsoft.com/azure/storage/blobs/access-tiers-overview?tabs=azure-portal).
 * Added `CopySourceTag` option for `UploadBlobFromURLOptions`
 * Added [FilterBlobs](https://learn.microsoft.com/rest/api/storageservices/find-blobs-by-tags-container) by tags API for container client.
-* Added `System` option to `ListContainersInclude` to allow listing of system containers. 
+* Added `System` option to `ListContainersInclude` to allow listing of system containers.
+* Updated the SAS Version to `2020-12-06` and added `Encryption Scope` to Account SAS, Blob SAS, and User Delegation SAS 
```

Comment:
```suggestion
* Updated the SAS Version to `2021-12-02` and added `Encryption Scope` to Account SAS, Service SAS, and User Delegation SAS 
```

Reply:
We should make sure we are sending the 2021-12-02 SAS version to the service, it should match the x-ms-version.